autogen.sh

```diff
@@ -2,7 +2,7 @@
 
 # Update autoconf scripts after a configure.ac change
 
-if [ ! -f .gitignore -o ! -f sbin/firehol.in ]
+if [ ! -f .gitignore -o ! -f sbin/firehol.in.in ]
```

Comment:
autogen.sh script should be oneliner autoreconf -ivf, nothing more.

I do not sure why you have these conditions, all files should be generated by make.